<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>:/Images/Technix_favicon.jpg</selectedon>
   </iconset>
  </property>
  <property name="whatsThis">
   <string>fgyfhgth</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="settingTabs">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="media">
      <attribute name="title">
       <string>Media</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Path:</string>
             </property>
             <property name="buddy">
              <cstring>path</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="path"/>
           </item>
           <item>
            <widget class="QPushButton" name="search">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="paths"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="add">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Delete">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="playback">
      <attribute name="title">
       <string>Playback</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="autoPlay">
             <property name="toolTip">
              <string>checked: autoplays next song, unchecked: it doesn't</string>
             </property>
             <property name="toolTipDuration">
              <number>0</number>
             </property>
             <property name="text">
              <string>Autoplay</string>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="continuePlayingPlaySelected">
             <property name="toolTip">
              <string>Checked: pressing play continues playing, unchecked: pressing play </string>
             </property>
             <property name="toolTipDuration">
              <number>0</number>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Continue playing/Play selected</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QComboBox" name="playbackOrder">
           <property name="toolTip">
            <string>Change playback order</string>
           </property>
           <property name="toolTipDuration">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Repeat Track</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Default</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Repeat Playlist</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Random</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoXfade">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>AutoXFade</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QCheckBox" name="autoFadeIn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>AutoFadeIn</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="FadeInLabel">
           <property name="text">
            <string>FadeIn-Time:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fadeInTime"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QCheckBox" name="autoFadeOut">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>AutoFadeOut</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="fadeOutLabel">
           <property name="text">
            <string>FadeOut-Time:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fadeOutTime"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advanced">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="reset">
         <property name="text">
          <string>Reset Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="ok">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>search</tabstop>
  <tabstop>add</tabstop>
  <tabstop>Delete</tabstop>
  <tabstop>path</tabstop>
  <tabstop>paths</tabstop>
  <tabstop>settingTabs</tabstop>
  <tabstop>autoPlay</tabstop>
  <tabstop>continuePlayingPlaySelected</tabstop>
  <tabstop>ok</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ok</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
